src/shared_modules/utils/stringHelper.h

```diff
@@ -97,6 +98,25 @@ namespace Utils
     {
         return leftTrim(rightTrim(str, args), args);
     }
+
+    static std::string toUpperCase(const std::string& str)
+    {
+        std::string temp{ str };
+        std::transform(std::begin(temp),
+                       std::end(temp),
+                       std::begin(temp),
+                       [](std::string::value_type character) { return std::toupper(character); });
+        return temp;
+    }
+
+    static bool startsWith(const std::string& str, const std::string& start)
+    {
+        if (str.empty() == false && str.length() >= start.length())
```

Comment:
!str.empty() instead of explicit boolean sentence

Reply:
will do